app/res/layout/question_widget_text_compact.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+          android:layout_width="0dp"
+          android:layout_weight="1"
+          android:layout_height="match_parent"
+          android:ellipsize="end"
+          android:maxLines="1"
+          android:scrollHorizontally="false"
+          android:text="QuestionWidget placeholder text: there should be a really big question text here, greater than the screen&apos;s size"/>
```

Comment:
Do you know what having this here does? I know it's present in `question_widget_text.xml` as well, but I don't know why

Reply:
It's just a placeholder. Doesn't really do anything. Should remove it from both places or use `tools:text` instead of `android:text` which would make it appear only in Android Studio xml preview.

Reply:
Ok let's do that